src/datastore/pg-store.ts

```diff
@@ -396,6 +396,80 @@ export class PgStore {
     });
   }
 
+  async getMetadataBlocks({ limit, offset }: { limit: number; offset: number }) {
+    return await this.sql.begin(async sql => {
+      // get block list
+      const { results: blocks, total: block_count } = await this.getBlocks({ limit, offset });
+      const blockHashValues = blocks
+        .map(block => `('\\x${block.index_block_hash.slice(2)}'::bytea)`)
```

Comment:
Parameter encoding for bytea types should already be handled automatically for 0x-prefix strings, see https://github.com/hirosystems/stacks-blockchain-api/blob/78d7d9c9f72db8ce6c59e0ea59a2579aceec014e/src/datastore/connection.ts#L24-L30